src/cloudcasting/validation.py

```diff
@@ -334,6 +345,13 @@ def validate(
         batch_limit (int | None, optional): Defaults to None. For testing purposes only.
     """
 
+    # Verify we can run the model forward
+    try:
+        model(np.zeros((1, 11, model.history_steps, 100, 100), dtype=np.float32))
```

Comment:
Some ML models will depend on the x and y shape of the input. This is 100 x 100 here but in the satellite data is 372 x 614. I think we should match the shape

Reply:
Done!